docs/getting-started.md

```diff
@@ -44,104 +35,86 @@ Replace the contents of `bin/<your-main-file>.ts` (where `your-main-file` by def
-                new ArnPrincipal(`arn:aws:iam::${YOUR_IAM_ACCOUNT}:user/user1`),  
-                new ArnPrincipal(`arn:aws:iam::${YOUR_IAM_ACCOUNT}:user/user2`)
-            ]
+Deploy the stack using the following command. This command will take roughly 20 minutes to complete.
 
-        });
-    }
-}
+```
+cdk deploy
 ```
 
-Replace the contents of `bin/<your-app-name>.ts` with the following:
-
-```typescript
+Congratulations! You have deployed your first EKS cluster with `cdk-eks-blueprint`. The above code will provision the following:
 
-import * as cdk from '@aws-cdk/core';
-import { EksBluepint, AddOns, ClusterAddOn, Team } from '@shapirov/cdk-eks-blueprint';
+- [x] A new Well-Architected VPC with both Public and Private subnets.
+- [x] A new Well-Architected EKS cluster in the region and account you specify.
+- [x] [Nginx](https://argoproj.github.io/argo-cd/) into your cluster to serve as a reverse proxy for your workloads. 
+- [x] [ArgoCD](https://argoproj.github.io/argo-cd/) into your cluster to support GitOps deployments. 
+- [x] [Calico](https://docs.projectcalico.org/getting-started/kubernetes/) into your cluster to support Network policies.
+- [x] [Metrics Server](https://github.com/kubernetes-sigs/metrics-server) into your cluster to support metrics collection.
+- [x] AWS and Kubernetes resources needed to to support [Cluster Autoscaler](https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html).
+- [x] AWS and Kubernetes resources needed to forward logs and metrics to [Container Insights](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/deploy-container-insights-EKS.html).
+- [x] AWS and Kubernetes resources needed to support [AWS Load Balancer Controller](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html).
 
-import { TeamAwesome } from '../teams/team-awesome';
+## Cluster Access
 
-const addOns: Array<ClusterAddOn> = [
-  new AddOns.ArgoCDAddOn,
-];
+Once the deploy completes, you will see output in your terminal window similar to the following:
 
-const teams: Array<Team> = [
-    new TeamAwesome
-]
+```bash
+Outputs:
+east-test-1.easttest1ClusterName8D8E5E5E = east-test-1
+east-test-1.easttest1ConfigCommand25ABB520 = aws eks update-kubeconfig --name east-test-1 --region us-east-1 --role-arn <ROLE_ARN>
+east-test-1.easttest1GetTokenCommand337FE3DD = aws eks get-token --cluster-name east-test-1 --region us-east-1 --role-arn <ROLE_ARN>
 
-const app = new cdk.App();
-const opts = {id: 'east-test-1', addOns, teams}
-new EksBlueprint(app, opts, {
-  env: {
-      account: 'XXXXXXXXXXXX',
-      region: 'us-east-2'
-  },
-});
+Stack ARN:
+arn:aws:cloudformation:us-east-1:115717706081:stack/east-test-1/e1b9e6a0-d5f6-11eb-8498-0a374cd00e27
 ```
 
-Deploy the stack using the following command
+To update your Kubernetes config for you new cluster, copy and run the `east-test-1.easttest1ConfigCommand25ABB520` (the middle command) command in your terminal. 
```

Comment:
the middle command is a bit confusing. I would add "the middle command in output". Also it makes sense to call out that this instruction is only for admins and only for the cluster owners: depending on the role ARN. We should call out the team/platform roles generated (or point to the teams doc).